src/core/translation/locales/translation.ts

```diff
@@ -0,0 +1,14 @@
+import languages from './languages.json';
+import skills from './skills.json';
+import socialCauses from './socialCauses.json';
+
+export function generateTranslationFile(theLanguage) {
+  //return Object.assign({}, skills, socialCauses, languages);
```

Comment:
remove comments please

Reply:
> remove comments please

Ok I removed all four comments in my latest push.